boards/arm/nrf52840_pca10056/Kconfig

```diff
@@ -6,4 +6,9 @@
 
 if BOARD_NRF52840_PCA10056
 
+config BOARD_HAS_DCDC
```

Comment:
Why not define this in the arch/arm/soc Kconfig and then enable it here?

Reply:
In the arch/arm/soc, I have defined a hidden option which is selected when this BOARD_HAS_DCDC is selected (which it is by default). This visible option provides the end-user of a board the option of disabling it.

The rationale is, even though the SoC supports DCDC, external components are required without which a SoC could be bricked due to missing components. Only upstream boards with BOARD_HAS_DCDC defined would select the hidden Kconfig option; this way the Chip is safe.

Reply:
Understood, makes sense to me